test/test_conversion_driver_esm1p5.py

```diff
@@ -135,30 +163,37 @@ def test_convert_fields_file_list_fail_excepted(mock_process_with_exception):
     fake_file_path = Path("fake_file")
 
     _, failed = esm1p5_convert.convert_fields_file_list(
-        [fake_file_path], "fake_nc_write_dir")
+        [fake_file_path], "fake_nc_write_dir", delete_ff=True)
```

Comment:
This also failed with the additional `delete_ff=True` flag arg.

Reply:
This one should be fixed now.